### PR TITLE
commercial.dcr.js hotfix

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?
- puts all exports on window and uses them if they exist

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -13,6 +17,18 @@ const initialised = new Promise((resolve) => {
...
 
 function init({ pubData, isInUsa }: { pubData?: PubData; isInUsa: boolean }) {
+	// *************** START commercial.dcr.js hotfix ***************
+	if (window?.guCmpHotFix?.initialised) {
+		return;
+	}
+
+	if (window) {
+		window.guCmpHotFix = {
+			initialised: true,
```
Comment (sndrs) [inline src/index.ts:27]:
```suggestion
			...window.guCmpHotFix,
			initialised: true,
```

File: src/index.ts [hunk 2]
```diff
@@ -35,12 +51,29 @@ function showPrivacyManager() {
...
+
+// export { onConsentChange } from './onConsentChange';
+
+const actualExports = {
+	cmp: { init, willShowPrivacyMessage, showPrivacyManager },
+	onConsentChange: actualOnConsentChange,
 };
 
-export { onConsentChange } from './onConsentChange';
+if (window) {
```
Comment (jeteve) [inline src/index.ts:68]: Shouldn't that be part of the init function to avoid assigning `guCmpHotFix` more than once?

Comment (mxdvl) [reply src/index.ts:69]: There's a `window.guCmpHotFix.initialised` value set on L28 to check it only gets initialised once.

Comment (jeteve) [reply src/index.ts:69]: I'm not sure that protects against double initialisation in the case the code is loaded as part of two different JS 
 bundles, as this window assignment is globally scoped?

Comment (mxdvl) [reply src/index.ts:69]: You're right, and that's why we spread using the spread operator below: `...window.guCmpHotFix`

**Pull request merged**